src/main/jbake/content/docs/components/metrics/index.adoc

```diff
@@ -1258,7 +1258,380 @@ A reset event occurs some time between 90000 and 210000; consequently, we will g
+----
+
+This request queries the rate for a counter named `total_requests` and specifies that 60 buckets be used. An array of
+numeric bucket data points is returned.
+
+[source,shell]
+.Fetch rate stats using bucketDuration parameter
+----
+curl -X GET http://server/hawkular/metrics/counters/total_requests/rate/stats?start=1235&end=6789&bucketDuration=60000 \
+-H "Content-Type: application/json" -H "Hawkular-Tenant: com.acme"
+----
+
+This request uses the `bucketDuration` parameter and specifies that each bucket is a minute wide.
+
+[source,shell]
+.Fetch rate stats that include percentiles
+----
+curl -X GET http://server/hawkular/metrics/total_requests/rate/stats?start=1235&end=6789&buckets=30&percentiles=75,90,99 \
+-H "Content-Type: application/json" -H "Hawkular-Tenant: com.acme"
+----
+
+You can also query for rates across multiple counter. The set of metrics to query is determined by using either tag
+filters or by specifying a list of metric names.
+
+[source,shell]
+.Fetch rate stats from multiple counters by name
+----
+curl -X GET http://server/hawkular/metrics/counters/rate/stats?start=12345&end=56789&buckets=100&metrics=C1,C2,C3
+----
+
+This request fetches rate data points from counters C1, C2, and C3. The only difference from previous examples is that each
+bucket will contain data points from multiple metrics.
+
+Next we use tag filters to select the set of metrics to query.
+
+[source,shell]
+.Fetch rate stats from counters using tag filters
+----
+curl -X GET http://server/hawkular/metrics/counters/rate/stats?start=1235&end=6789&buckets=30&tags=hostname:server1 \
+-H "Content-Type: application/json" -H "Hawkular-Tenant: com.acme"
+----
+
+==== Availability Bucket Data Points
+Availability bucket data points are used with availability metrics. When data points are grouped into a bucket, several
+aggregation functions are applied to produce a data point that consists of several of statistics.
+
+[source,javascript]
+.Availability bucket data point
+----
+{
+  "start": 12345,
+  "end": 6789,
+  "empty": false,
+  "downtimeDuration": 29311,
+  "lastDowntime": 12367,
+  "uptimeRatio": 0.78,
+  "downtimeCount": 12
+}
+----
+
+`start` and `end` correspond to the bucket's start and end times respectively.
+
+`empty` is a boolean flag that indicates whether or not the bucket has any data points in it. We will see an example of
+an empty bucket next.
+
+`downtimeDuration` is the total time in milliseconds that the metric was reported down. Note that this is the total
+time within the bucket's stat and end times.
+
+`lastDowntime` is the last time within the bucket's time range that the metric was reported down. The value is in
+milliseconds.
+
+`uptimeRatio` is basically a percentage of the time for the duration of the bucket that the metric is up. The value
+will be a floating point number between zero and one.
+
+`downtimeCount` is the number of times within the duration of the bucket that the metric was reported down.
```

Comment:
Not exactly. If the resource was reported down twice, then up, then down again, downtimeCount equals 2. It's the number of downtime periods, not reports.